From dd6328cb8c4a90f99f48bffdcbbf3b5134cf4c2b Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Tue, 11 Oct 2016 12:49:42 +0200
Subject: [PATCH 1/3] check-accelerated: Fix warnings on startup

The Wayland session is now named "gnome" not "gnome-wayland", so don't
try to run the X11-specific code when under Wayland.

https://bugzilla.gnome.org/show_bug.cgi?id=756914
---
 tools/gnome-session-check-accelerated.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/gnome-session-check-accelerated.c b/tools/gnome-session-check-accelerated.c
index caade1c..810478c 100644
--- a/tools/gnome-session-check-accelerated.c
+++ b/tools/gnome-session-check-accelerated.c
@@ -107,7 +107,7 @@ main (int argc, char **argv)
          * case, and it currently requires X. Until we have that working, just always
          * assume wayland will work
          */
-        if (g_strcmp0 (g_getenv ("XDG_SESSION_DESKTOP"), "gnome-wayland") == 0) {
+        if (g_strcmp0 (g_getenv ("XDG_SESSION_TYPE"), "x11") != 0) {
                 return 0;
         }
 
-- 
2.9.3

